<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1061</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="toolbarLayout">
    <item>
     <widget class="QPushButton" name="attachButton">
      <property name="text">
       <string>DeAttach</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stepIntoButton">
      <property name="text">
       <string>StepInto</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>401</width>
     <height>821</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableView" name="registersTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registers</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputRegistersTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input Registers</string>
    </property>
   </widget>
   <widget class="QTableView" name="inputRegistersTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QListView" name="dissamblerListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>631</width>
     <height>501</height>
    </rect>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
